components/usage/pkg/db/cost_center.go

```diff
@@ -158,19 +158,19 @@ func (c *CostCenterManager) UpdateCostCenter(ctx context.Context, costCenter Cos
 
 func (c *CostCenterManager) ComputeInvoiceUsageRecord(ctx context.Context, attributionID AttributionID) (*Usage, error) {
 	now := time.Now()
-	summary, err := GetUsageSummary(ctx, c.conn, attributionID, now, now, false)
+	creditCents, err := GetBalance(ctx, c.conn, attributionID)
 	if err != nil {
 		return nil, err
 	}
-	if summary.CreditCentsBalanceAtEnd.ToCredits() <= 0 {
+	if creditCents.ToCredits() <= 0 {
 		// account has no debt, do nothing
 		return nil, nil
 	}
```

Comment:
While we're here, we could also change the logic to do `creditCents <= 0` because the Credits conversion only multiplies by a 100, which won't change the polarity